src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -705,6 +705,22 @@ public BlackCard getCurrentBlackCard(Long userId) {
     return user.getActiveGame().getBlackCard();
   }
 
+    /**
+     * Gets the active game of a user, but throws 404 if the user has no active game or no black card selected yet
+     * @param userId: UserID of the user that we want the current black card of
+     * @return activeGame (or 404)
+     */
+
+    public Game getActiveGame(Long userId) {
+        User user = getUserById(userId);
+        // check if user has active game, or a black card chosen respectively
+        if (user.getActiveGame() == null || user.getActiveGame().getBlackCard() == null) {
```

Comment:
The second condition is never null, since the black card get's assigned to the game when creating one.

Reply:
done